support/signal.c

```diff
@@ -298,6 +299,15 @@ default_handler (int signum)
 
 static pthread_mutex_t signals_mutex = PTHREAD_MUTEX_INITIALIZER;
 
+#if HAS_PTHREAD_PRIO_INHERIT
+__attribute__((constructor))
+__attribute__((unused))
```

Comment:
1. This is `G_GNUC_UNUSED`
2. We should make `G_GNUC_CONSTRUCTOR`

Reply:
Or something that works with msvc also? There is a way.